src/pages/App.tsx

```diff
@@ -114,6 +115,8 @@ export default function App() {
   useAnalyticsReporter()
   initializeAnalytics()
 
+  useSyncChainQuery()
```

Comment:
Can this still be in the network selector so that it's not global, but grouped with like logic?

Reply:
yeah i can move it back